src/AppKit/Enums.cs

```diff
@@ -2898,4 +2898,20 @@ public enum NSColorSystemEffect : long {
 		Disabled,
 		Rollover,
 	}
+
+	[Mac (10,14, onlyOn64: true)]
+	[Native]
+	public enum NSWorkspaceAuthorizationType : long  {
+		CreateSymbolicLink,
+		SetAttributes,
+		ReplaceFile,
+	}
+
+	[Mac (10,14, onlyOn64: true)]
+	[Native]
+	public enum NSWorkspaceAuthorizationError : ulong  {
+		Invalid = 67328,
+		Minimum = 67328,
+		Maximum = 67455,
```

Comment:
Don't add `Minimum` and `Maximum` since they could change in future versions - and that's a breaking change for us